test/e2e/specs/editor/various/block-switcher.spec.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	blockSwitcher: async ( { page }, use ) => {
+		await use( new blockSwitcher( { page } ) );
+	},
+} );
+
+test.describe( 'Block Switcher', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'Should show the expected block transforms on the list block when the blocks are removed', async ( {
+		page,
+		blockSwitcher,
+		pageUtils,
+	} ) => {
+		// Insert a list block.
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '- List content' );
+
+		await page.keyboard.press( 'ArrowUp' );
+		await pageUtils.pressKeys( 'alt+F10' );
+
+		// Verify the block switcher exists.
+		expect( page.locator( 'role=button[name="List"i]' ) ).toBeTruthy();
```

Comment:
A locator is always truthy. Let's try `toBeVisible()` instead? We can also scope it under the block toolbar.

```suggestion
		const blockToolbar = page.getByRole( 'toolbar', { name: 'Block tools' } );
		await expect( blockToolbar.getByRole( 'button', { name: 'List' } ) ) ).toBeVisible();
```